packages/wdio-cli/src/constants.js

```diff
@@ -77,6 +77,7 @@ export const SUPPORTED_PACKAGES = {
         { name: 'appium', value: '@wdio/appium-service$--$appium' },
         { name: 'firefox-profile', value: '@wdio/firefox-profile-service$--$firefox-profile' },
         { name: 'crossbrowsertesting', value: '@wdio/crossbrowsertesting-service$--$crossbrowsertesting' },
+        { name: 'slack', value: '@wdio/slack-service$--$slack' },
```

Comment:
This is not an internal package as it comes from you (a 3rd party contributor). Therefor let's move it into the `external` section.